hack/versions/pkg/schema/comment.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schema
+
+import (
+	"bytes"
+	"go/ast"
+	"go/parser"
+	"go/printer"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+const releasedComment = `// !!! WARNING !!! This config version is already released, please DO NOT MODIFY the structs in this file.`
+const unreleasedComment = `// This config version is not yet released, it is SAFE TO MODIFY the structs in this file.`
+
+// historicalVersionComments is used to recognize whether an existing comment is a "release comment" or not.
+// If you want to change a (say releasedComment) comment historically on all files, then:
+// 1.) add the current version to `historicalVersionComments`
```

Comment:
shd the order be preserved. i. e `historicalVersionComments[0]`  is old release comment and 1 is unreleasedComment?

Reply:
order doesn't matter!